### DISCUSSION TITLE
If I want to set the amplitude of the source, how should I operate ?

### DISCUSSION THREAD

**Discussion by xungu1** [kc-ml2/meent#69] [Q&A] [answered]
I want to change the amplitude to see the change of the electromagnetic field after the simulation of the grating by meent, or as I think, the amplitude is only the value of the electromagnetic field is scaled ? Another question is how do I see the phase change ? For example, I simulate a one-dimensional grating, how should I view its phase change ?

**Comment by yonghakim** [comment, marked as answer]
Hi,

1) the amplitude of incident light is set to 1. 

2) Field returned by Meent is array of complex numbers. I think you can use `numpy.angle()` to find phase. Is this what you are looking for?
